test/extended/setup.sh

```diff
@@ -16,7 +16,40 @@ function os::test::extended::focus {
   fi
 }
 
-# Launches an extended server for OpenShift
+# Run conformance test suite.
+function os::test::extended::conformance {
+  function join { local IFS="$1"; shift; echo "$*"; }
+
+  parallel_only=( "${CONFORMANCE_TESTS[@]}" )
+  parallel_exclude=( "${EXCLUDED_TESTS[@]}" "${SERIAL_TESTS[@]}" )
+  serial_only=( "${SERIAL_TESTS[@]}" )
+  serial_exclude=( "${EXCLUDED_TESTS[@]}" )
+
+  pf=$(join '|' "${parallel_only[@]}")
+  ps=$(join '|' "${parallel_exclude[@]}")
+  sf=$(join '|' "${serial_only[@]}")
+  ss=$(join '|' "${serial_exclude[@]}")
+
+  echo "[INFO] Running the following tests:"
+  TEST_REPORT_DIR= TEST_OUTPUT_QUIET=true ${EXTENDEDTEST} "--ginkgo.focus=${pf}" "--ginkgo.skip=${ps}" --ginkgo.dryRun --ginkgo.noColor | grep ok | grep -v skip | cut -c 20- | sort
```

Comment:
While someone's mucking with this we need to either:
 - remove this
 - make it handle the case where no tests are selected
 - make it sane code and not rely on a 10x pipe so above step is easier